sdks/python/README.md

```diff
@@ -2,6 +2,8 @@
 
 This is the Python SDK for Argo Workflows.
 
+⚠️ This SDK is deprecated and will be removed in version 3.7 of Argo Workflows. It is recommended that you migrate to using [hera](https://github.com/argoproj-labs/hera).
```

Comment:
```suggestion
> [!WARNING]
> This SDK is deprecated and will be removed in version 3.7 of Argo Workflows.
> We recommend migrating to [Hera](https://github.com/argoproj-labs/hera).
```

- We can use fancy [GH-style admonitions](https://github.com/orgs/community/discussions/16925) here to make this more prominent.
- prefer 1 sentence per line of markdown, per [style guide](https://argo-workflows.readthedocs.io/en/release-3.5/doc-changes/)
- Use active voice, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)
- use either capital "Hera" as a proper noun or backticked `hera` as the package name

Reply:
I had selected to use a readable glyph here as this file is likely to be read using something other than Github as a renderer.

Reply:
Yea if this is used for Pypi, it probably wouldn't work. But that can be addressed as part of #13704